### PR TITLE
Switch to using GH reviews instead of approve labels

### PR DESCRIPTION
**Pull request by mads-hartmann**
Leaving 

## Description

Switch Tide to use GitHub Reviews instead of lgtm and approve labels for finding PRs that it should merge. This is part of the work require to switch fully over to GitHub Reviews ([epic](https://github.com/gitpod-io/ops/issues/838)).

## Related Issue(s)

Fixes https://github.com/gitpod-io/ops/issues/842

## How to test

Once merged we can check the Tide query to verify it finds the correct PRs (this is the config already used in ops, so I'm pretty confident it works)

## Release Notes

```release-note
NONE
```

## Documentation

N/A

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **mads-hartmann** after the PR has been reviewed.

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitbot).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitbot/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by mads-hartmann** [comment]
Added the do-not-merge/hold label as we should not merge this until we're ready to make the switch fully.

**Review by ArthurSens — APPROVED** [review]
I'll leave it to you to decide the best moment to merge :)

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitbot/pull/65#pullrequestreview-870745360" title="Approved">ArthurSens</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitbot).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitbot/blob/main/OWNERS)~~ [ArthurSens]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 2c772b029ede64f219cde26b0fd3a3807103195c</details>

**Comment by roboquat** [comment]
@mads-hartmann: Updated the `config` configmap in namespace `prow` at cluster `default` using the following files:
 - key `config.yaml` using file `config/config.yaml`

<details>

In response to [this](https://github.com/gitpod-io/gitbot/pull/65):

>Leaving 
>
>## Description
>
>Switch Tide to use GitHub Reviews instead of lgtm and approve labels for finding PRs that it should merge. This is part of the work require to switch fully over to GitHub Reviews ([epic](https://github.com/gitpod-io/ops/issues/838)).
>
>## Related Issue(s)
>
>Fixes https://github.com/gitpod-io/ops/issues/842
>
>## How to test
>
>
>Once merged we can check the Tide query to verify it finds the correct PRs (this is the config already used in ops, so I'm pretty confident it works)
>
>## Release Notes
>
>```release-note
>NONE
>```
>
>## Documentation
>
>N/A


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
